x-pack/legacy/plugins/actions/server/actions_config.ts

```diff
@@ -47,20 +49,24 @@ function doesValueWhitelistAnyHostname(whitelistedHostname: string): boolean {
 function isWhitelisted({ whitelistedHosts }: ActionsKibanaConfig, hostname: string): boolean {
   return (
     Array.isArray(whitelistedHosts) &&
-    fromNullable(
-      whitelistedHosts.find(
-        whitelistedHostname =>
-          doesValueWhitelistAnyHostname(whitelistedHostname) || whitelistedHostname === hostname
+    isSome(
+      fromNullable(
+        whitelistedHosts.find(
+          whitelistedHostname =>
+            doesValueWhitelistAnyHostname(whitelistedHostname) || whitelistedHostname === hostname
+        )
       )
-    ).isSome()
+    )
   );
 }
 
 function isWhitelistedHostnameInUri(config: ActionsKibanaConfig, uri: string): boolean {
-  return tryCatch(() => new URL(uri))
-    .map(url => url.hostname)
-    .mapNullable(hostname => isWhitelisted(config, hostname))
-    .getOrElse(false);
+  return pipe(
+    tryCatch(() => new URL(uri)),
+    map(url => url.hostname),
+    mapNullable(hostname => isWhitelisted(config, hostname)),
+    fold(constant(false), identity)
```

Comment:
Can we place change this to:
```
getOrElse(() => false)
```

It reads a little clearer and is still correct. :)